includes/definitions/discovery/supermicro-bmc.yaml

```diff
@@ -0,0 +1,97 @@
+mib: ATEN-IPMI-MIB
+modules:
+    os:
+        version:
+            - ATEN-IPMI-MIB::bmcMajorVesion.0
+            - ATEN-IPMI-MIB::bmcMinorVesion.0
+        version_template: "FW {{ ATEN-IPMI-MIB::bmcMajorVesion.0 }}.{{ ATEN-IPMI-MIB::bmcMinorVesion.0 }}"
+        features: ATEN-IPMI-MIB::biosVesion.0
```

Comment:
Typo ?
```biosVersion OBJECT-TYPE
    SYNTAX      OCTET STRING (SIZE(32))
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION
            "BIOS version of the controller."
    ::= { controllerEntry 6 }```

Reply:
(yes, the mib is full of errors, but this one they typed it correctly so you also have to type it right :D )

Reply:
> Typo ?

hi @PipoCanaja 
please read this block from MIB

```
bmcMajorVesion  OBJECT-TYPE
    SYNTAX      Integer32 (0..255)
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION
            "BMC Major Vesion"
    ::= { boardinfo 1 }

bmcMinorVesion  OBJECT-TYPE
    SYNTAX      Integer32 (0..255)
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION
            "BMC Minor Vesion"
    ::= { boardinfo 2 }
```

do you see ???

Reply:
> Typo ?

and again ...
```
discovery:
    -
        sysObjectID: .1.3.6.1.4.1.8072.3.2.10
        snmpget:
            oid: ATEN-IPMI-MIB::bmcMajorVesion.0
```

Reply:
Yes but did you read my comment : The block I mentionned is `biosVersion` and you wrote it `biosVesion`. Please correct biosVesion to biosVersion and I'll be more than happy to close this part of the review :)

Reply:
> biosVersion

please mr @PipoCanaja 
it is NOT typo

```
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::controllerTable
ATEN-IPMI-MIB::controllerTable = No Such Object available on this agent at this OID
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::controllerEntry
ATEN-IPMI-MIB::controllerEntry = No Such Object available on this agent at this OID
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::biosVersion
ATEN-IPMI-MIB::biosVersion = No Such Object available on this agent at this OID
```

and here it is ! 
without "R" letter !!!
what else proof do you need ?
```
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::biosVesion
ATEN-IPMI-MIB::biosVesion.0 = STRING: "3.3             "
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::bmcMajorVesion
ATEN-IPMI-MIB::bmcMajorVesion.0 = INTEGER: 3
librenms@nms:~$ snmpwalk -v2c -c XXX fd00:25:76:0:225:90ff:fe86:dc49 -M mibs:mibs/supermicro -m ATEN-IPMI-MIB ATEN-IPMI-MIB::bmcMinorVesion
ATEN-IPMI-MIB::bmcMinorVesion.0 = INTEGER: 88
```

Reply:
The mib has `biosVersion` as I showed you.

Line 541 of the MIB file : 
```
ControllerEntry ::=
    SEQUENCE {
        controllerNumber            Integer32,
        controllerProductName       OCTET STRING,
        serial                      OCTET STRING,
        package                     OCTET STRING,
        fwVersion                   OCTET STRING,
        biosVersion                 OCTET STRING,
        bootBlockVersion            OCTET STRING,
        batteryStatus               Integer32,
        pcieLocation                Integer32,
        pcieSlot                    Integer32
    }
```

and line 597

```
biosVersion OBJECT-TYPE
    SYNTAX      OCTET STRING (SIZE(32))
    MAX-ACCESS  read-only
    STATUS      current
    DESCRIPTION
            "BIOS version of the controller."
    ::= { controllerEntry 6 }
```

So if you read correctly my comment, it seems that **I need you to show me how stupid this MIB file is**, as it happens that this MIB has both `biosVersion` and `biosVesion`. So you could start by explaining me that, instead of being rude ... 

I spend my free time trying to review this PR, so yes, you have to explain the reviewers like me that this PR is ready for merge. Not me having to convince you of anything. I am asking questions, you help me understand. If you make my review easy, I can do it quickly, your PR gets merge. If you are rude, I waste time, you waste time, you make me wasting even more time writing this paragraph, and at the end of it, I don't have any more time to Approuve and Merge this PR. 
It is **that** simple ...

Reply:
So you could start by explaining me that, instead of being rude ...

far from rude
English is not my native language, so if you feel that i am rude, then sorry

and after 40 successful PR, mostly adding devices to LNMS ...
and attaching a successful YAML tests ...
please allow me to to ask

>I don't have any more time to Approuve and Merge this PR
so, if i understand correctly ? you don't want to merge valid, 8 test passed PR ??? for new device ?